rest-fetcher/src/test/java/com/ackee/versionupdatehandler/RestVersionFetcherTest.kt

```diff
@@ -0,0 +1,94 @@
+package com.ackee.versionupdatehandler
+
+import com.google.common.truth.Truth
+import com.google.gson.JsonObject
+import com.google.gson.JsonPrimitive
+import kotlinx.coroutines.runBlocking
+import okhttp3.mockwebserver.MockResponse
+import okhttp3.mockwebserver.MockWebServer
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+
+/**
+ * Tests for [RestVersionFetcher]
+ *
+ * @author Eduard Ablekimov [eduard.ablekimov@ackee.cz]
```

Comment:
can be removed as well. remove it everywhere pls

Reply:
5ff6590de61b394853088024f80a06efab417818